### ISSUE TITLE
Update Permalink 'Preview' to 'View' on published posts

### ISSUE THREAD

**Issue by garrett-eclipse** [WordPress/gutenberg#14206]
**Is your feature request related to a problem? Please describe.**
On a published post the 'Preview' in the top nav previews the current edits if any while the 'Preview' link in the Permalink modules will always link to the published page and not a preview. 

<img width="275" alt="screen shot 2019-03-03 at 1 52 53 am" src="https://user-images.githubusercontent.com/8726005/53693669-9e136b80-3d58-11e9-8c4e-886b33b32585.png">

**Describe the solution you'd like**
Can the 'Preview' verbiage in the Permalink module be updated to 'View' on published posts.

**Describe alternatives you've considered**
I've also thought it would be nice to have a View button in the topbar along with Preview on published posts to avoid having to find it in the Permalinks accordion as new users are unaware of how to view the post. When the wp-admin is available there's at least the link there, but when in full-screen mode this isn't available.

Thank you

**Comment by youknowriad** [comment]
I think the idea is that it means "preview of the link" but I may be wrong.

**Comment by garrett-eclipse** [comment]
Thanks @youknowriad 

I think my main gripe with the verbiage there is the 'Preview' button in the top actually generates a preview of the page and any changes appending `?preview_id=##&preview_nonce=XXXX&preview=true` to the URL. Whereas the 'Permalink' provides the actual URL which takes you to the public page in it's current state (without any of your non-saved edits). As both of these are labelled as 'Preview' but do completely different actions I feel they should be distinctly defined by a slight change in verbiage.

When in draft the issue still pertains as they do two separate actions but in that context I'm ok with the Permalink stating Preview.

Aside from changing the Permalink section to say 'View' the other thought was to update the main Preview button that generates the preview with current changes to state 'Preview Changes' so it's clear that it would take you to the preview but also account for any changes you've made to the page without saving.

Hope that makes a bit of sense.
Thanks

**Comment by michelleweber** [comment]
Seems like "View published page" or "View page" for the permalink and "Preview changes" in the nav would be a clearer way to go.

**Comment by sarahmonster** [comment]
Agreed that this could use better terminology to reflect the different statuses. For reference, here's a visualisation of the current situation:

![current-situation](https://user-images.githubusercontent.com/376315/57320817-a53a5b00-70b4-11e9-9451-a4dc0043545a.png)

The problem here is that we use "preview" for two different links, that may potentially show different things, depending on whether the user has unpublished changes staged or not.

Since these two links go to different views, they should use different terminology so it's clear which is which.

## Suggested solution

Whilst I love the idea of "Preview changes" in the nav, I think it'll just be too wordy/long for the available space, especially with translations coming into the mix. Instead, let's just change the text for the permalink link. 

"View published page" is definitely the clearest and most explicit option here:


![alternate-solution](https://user-images.githubusercontent.com/376315/57321020-2c87ce80-70b5-11e9-8fd2-98191453a575.png)


...but it's also a bit wordy, and we might be able to make do with just "view page" instead:

![suggested-solution](https://user-images.githubusercontent.com/376315/57320906-e9c5f680-70b4-11e9-951f-20ac41f49dbb.png)

**Comment by garrett-eclipse** [comment]
Thanks @sarahmonster I appreciate the illustrations there, they definitely convey the issue much better than words.

I agree we probably can't change the Preview button version so cleaning up the link verbiage is the best approach. View or View page works nicely, although in most cases we're dealing with a post or CPT so maybe 'page' should be the name of the current CPT or just drop it to use 'View'.

**Comment by sarahmonster** [comment]
My designer brain always wants pictures to go with everything. 😉 

Agreed on the custom post type issue! Ideally, we'd swap out "page" in the above suggestion for "post" or the custom post type name, in the same way that post notifications do:

<img width="668" alt="Screenshot 2019-05-08 at 00 23 17" src="https://user-images.githubusercontent.com/376315/57339000-789e3780-70e4-11e9-8fdd-e8fad99cfcd6.png">

**Comment by garrett-eclipse** [comment]
Awesome, thanks @sarahmonster I feel we're on the same page. Cheers

**Comment by mapk** [comment]
The solution here looks great! Let's update the Permalink wording to "View [page type]." I'm removing the `needs design feedback`. Thanks all!
